data/meta/tokens/1/0x48fF31bBbD8Ab553Ebe7cBD84e1eA3dBa8f54957.json

```diff
@@ -0,0 +1,50 @@
+{
+  "$schema": "token",
+  "name": "Curve sdAG-AG Pool",
+  "symbol": "sdAGAG-f",
+  "description": "This token represents a Curve factory liquidity pool. Holders earn fees from users trading in the pool, and can also deposit the LP to Curve's gauges to earn CRV emissions. This pool contains ANGLE and sdANGLE.",
```

Comment:
This pool contains ANGLE and sdANGLE, a tokenized veANGLE from StakeDAO.

Reply:
fixed